main.cc

```diff
@@ -1108,9 +1108,9 @@ int main(int ac, char** av) {
             }
 
             static redis_service redis;
-            if (cfg->enable_redis_protocol()) {
+            if (cfg->redis_port()) {
```

Comment:
But, doesn't this mean that Redis will now be enabled by default, because redis_port has a default?
Because the Redis support is experimental, we don't want to enable it by default.
So we either need redis_port to default to 0, or keep a separate enable-redis option...

Reply:
I think it's better to keep a separate enable-redis option.